popup.js

```diff
@@ -0,0 +1,72 @@
+// Define image and header data objects
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224435892-beada9fd-2fde-42a5-9d38-8a856ddb6bfc.png)

**[Optional]**


- [ ] It would be nice if you remove on the Setup sub section from your readme file because it doesn't make any sense if we see two sections with same name and same information 🤔